surfer/viz.py

```diff
@@ -998,7 +996,7 @@ def add_data(self, array, min=None, max=None, thresh=None,
             or a custom look up table (an n x 4 array coded with RBGA values
             between 0 and 255).
         alpha : float in [0, 1]
-            alpha level to control opacity
+            alpha level to control opacity of the surface.
```

Comment:
This is a little confusing since it controls the opacity of the statistical overlay w/r/t whatever is plotted under it, not the mesh itself (which I assume most would think of as the "surface") right?